backend/app/api/v1/commons/utils.py

```diff
@@ -90,9 +90,11 @@ def build_sort_terms(sort_string: str) -> list[dict[str, str]]:
     return sort_terms
 
 
-def buildAggregateQuery():
+def buildAggregateQuery(category):
     aggregate = {}
-    for x, y in constants.FIELD_CONSTANT_DICT.items():
+    category_data = getattr(constants, category)
+    # category_data = constants[category]
```

Comment:
That's an interesting change. Was that just to avoid the extra imports? Anyway, you should delete the commented line now.